lib/src/indicator.dart

```diff
@@ -107,6 +125,23 @@ class CirclePainter extends CustomPainter {
     canvas.drawCircle(Offset(selectedX, radius), radius, _selectedPaint);
   }
 
+  void paintOval(Canvas canvas, Size size) {
```

Comment:
The `size` and `padding` parameters of `PageIndicatorContainer` should be more intuitive in UI.

Reply:
Now Size is assumed to be diameter for circle
and width for RoundRectangle for library. Padding is same as before

Reply:
Thanks for PR.